server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -66,28 +66,12 @@ export class AssetController {
         if (!savedAsset) {
           return;
         }
-        if (uploadFiles.thumbnailData != null) {
-          const assetWithThumbnail = await this.assetService.updateThumbnailInfo(
-            savedAsset,
-            uploadFiles.thumbnailData[0].path,
-          );
-
-          await this.assetUploadedQueue.add(
-            'asset-uploaded',
-            { asset: assetWithThumbnail, fileName: file.originalname, fileSize: file.size, hasThumbnail: true },
-            { jobId: savedAsset.id },
-          );
-
-          this.wsCommunicateionGateway.server
-            .to(savedAsset.userId)
-            .emit('on_upload_success', JSON.stringify(assetWithThumbnail));
-        } else {
-          await this.assetUploadedQueue.add(
-            'asset-uploaded',
-            { asset: savedAsset, fileName: file.originalname, fileSize: file.size, hasThumbnail: false },
-            { jobId: savedAsset.id },
-          );
-        }
+
+        await this.assetUploadedQueue.add(
+          'asset-uploaded',
+          { asset: savedAsset, fileName: file.originalname, fileSize: file.size },
```

Comment:
The Job payload should probably be a shared interface `IAssetUploadedJob`. We could do it now creating a `libs/job` and configure it as a shared lib with something like `@immich/job` or `@app/job` 😃